explorer/explorerroutes.go

```diff
@@ -218,6 +218,69 @@ func (exp *explorerUI) Home(w http.ResponseWriter, r *http.Request) {
 	io.WriteString(w, str)
 }
 
+// Next reduction is the page handler for the "/nextreduction" path.
+func (exp *explorerUI) NextReduction(w http.ResponseWriter, r *http.Request) {
+	// Safely retrieve the current inventory pointer.
+	inv := exp.MempoolInventory()
+	// Lock the shared inventory struct from change (e.g. in MempoolMonitor).
+	inv.RLock()
+	exp.pageData.RLock()
+	// Get fiat conversions if available
+	homeInfo := exp.pageData.HomeInfo
+	var conversions *homeConversions
+	height, err := exp.dataSource.GetHeight()
+	if err != nil {
+		log.Errorf("GetHeight failed: %v", err)
+		exp.StatusPage(w, defaultErrorCode, defaultErrorMessage, "",
+			ExpStatusError)
+		return
+	}
+	blocks := exp.dataSource.GetExplorerBlocks(int(height), int(height)-8)
+	var bestBlock *types.BlockBasic
+	if blocks == nil {
+		bestBlock = new(types.BlockBasic)
+	} else {
+		bestBlock = blocks[0]
+	}
+	xcBot := exp.xcBot
+	if xcBot != nil {
+		conversions = &homeConversions{
```

Comment:
`ExchangeRate, StakeDiff and CoinSupply` are not being used.  You may want to create a new struct.